### PR TITLE
fixes case that parseNotableEventsRaw not setup yet

### PR DESCRIPTION
**Pull request by meirwah**

### CODE REVIEW

File: Integrations/integration-SplunkPy.yml [hunk 1]
```diff
@@ -1,8 +1,8 @@
 commonfields:
-  id: SplunkPy
+  id: SplunkPyx
   version: -1
-name: SplunkPy
-display: SplunkPy
+name: SplunkPyx
```
Comment (orius123) [inline Integrations/integration-SplunkPy.yml]: Change back

File: Integrations/integration-SplunkPy.yml [hunk 2]
```diff
@@ -168,11 +168,12 @@ script:
         incident["occurred"] = event["_time"]
         incident["rawJSON"] = json.dumps(event)
         labels = []
-        isParseNotableEventsRaw = demisto.params()['parseNotableEventsRaw']
-        if isParseNotableEventsRaw:
-            rawDict = rawToDict(event['_raw'])
-            for rawKey in rawDict:
-                labels.append({'type':rawKey,'value':rawDict[rawKey]})
+        if demisto.get(demisto.params(),'parseNotableEventsRaw'):
```
Comment (orius123) [inline Integrations/integration-SplunkPy.yml]: I think you can just add a `default` for `get` function

Comment (meirwah) [reply Integrations/integration-SplunkPy.yml]: is it supported?
Anyway it doesn't matter since default is false

**Comment by meirwah** [comment]
@orius123 plz review so we can release content

**Pull request merged**